qiita_db/metadata_template.py

```diff
@@ -2055,13 +2058,16 @@ def create_qiime_mapping_file(self, prep_template_fp):
         return filepath
 
 
-def load_template_to_dataframe(fn):
+def load_template_to_dataframe(fn, strip_values=True):
     """Load a sample or a prep template into a data frame
 
     Parameters
     ----------
     fn : str
         filename of the template to load
+    strip_values : bool, optional
```

Comment:
I would probably change this to `strip_whitespace`, but that's just a personal opinion and non-blocking to this PR.